R/utils.R

```diff
@@ -171,7 +171,8 @@ enw_example <- function(type = c(
 #' with error handling
 #'
 #' @param dates A vector-like input, which the function attempts
-#' to coerce via [data.table::as.IDate()].
+#' to coerce via [data.table::as.IDate()]. Defaults to NULL (i.e. an
```

Comment:
I don't see a default argument specified below; also confusing to suggest NULL and an empty vector are the same. Intent is that the default argument (NULL) results in returning an empty vector?

Reply:
See [`4b9fb24` (#402)](https://github.com/epinowcast/epinowcast/pull/402/commits/4b9fb248c2441db63b01548749e003c3aaa62fc2) and [`a7c5e8c` (#402)](https://github.com/epinowcast/epinowcast/pull/402/commits/a7c5e8cc8353f9333964a67d9c87127020ecd34f)